README.md

```diff
@@ -15,6 +15,35 @@ __Jump To:__
 
 ## Installation
 
+### Supported Architectures
```

Comment:
Please move the "**Supported Architectures**" section outside of "**Installation**" section. It should be on it own. Keep it at the top.